src/components/app/CompareHome.js

```diff
@@ -10,7 +10,7 @@ import { AppHeader } from './AppHeader';
 import { changeProfilesToCompare } from 'firefox-profiler/actions/app';
 import explicitConnect from 'firefox-profiler/utils/connect';
 import type { ConnectedProps } from 'firefox-profiler/utils/connect';
-
+import { Localized } from '@fluent/react';
```

Comment:
nit: we can move this import under `import React, { PureComponent } from 'react';`. (because it's an external library import)